ground/src/main/java/com/google/android/ground/ui/home/locationofinterestdetails/SubmissionListViewModel.java

```diff
@@ -75,9 +75,12 @@ private Single<List<Submission>> getSubmissions(SubmissionListRequest req) {
       // TODO(#354): Show message or special treatment for layer with no task.
       return Single.just(List.of());
     }
-    return submissionRepository
-        .getSubmissions(req.surveyId, req.locationOfInterestId, req.taskId.get())
-        .onErrorResumeNext(this::onGetSubmissionsError);
+
+    //    return submissionRepository
```

Comment:
I think you mean to delete this?

Reply:
I considered deleting it, but we might need this class for displaying the submission list in the future. So, I've commented it out and left a TODO to replace the code with appropriate kotlin coroutine in the future. 

Please let me know if you feel too strongly about deleting this code.

Reply:
Ah sorry I missed it. Maybe more the TODO up so that the example can be considered part of the TODO?

Reply:
Done